### PR TITLE
Update for edit

### PR DESCRIPTION
**Pull request by scott062**
FE: https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2692
API: https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/839
Mock: https://github.com/MetaPhase-Consulting/mock-fsbid/pull/401

- Users can create NEW remarks and inserts 
- Users CANNOT modify existing remarks/inserts EXCEPT to mark as inactive or active

### CODE REVIEW

File: src/Components/AdministratorPage/EditRemark/EditRemark.jsx [hunk 1]
```diff
@@ -31,22 +31,37 @@ const EditRemark = (props) => {
...
+        create_id,
+      }))
+      :
+      dispatch(saveRemark({
+        rmrkInsertionList,
+        rmrkCategory,
+        longDescription,
+        shortDescription,
+        activeIndicator,
+      }))
```
Comment (pveland621) [inline src/Components/AdministratorPage/EditRemark/EditRemark.jsx:63]: so if this isn't an edit and presumably we're creating a new remark, shouldn't we send back an `update_date` at creation? or is that handled in the backend where it just fills it out

File: src/actions/remark.js [hunk 2]
```diff
@@ -124,3 +124,26 @@ export function saveRemark(props) {
...
+    dispatch(saveAdminRemarkIsLoading(true));
+    dispatch(saveAdminRemarkHasErrored(false));
+    api().put(`/fsbid/remark/${props.seq_num}/`, props)
+      .then(() => {
+        batch(() => {
+          dispatch(saveAdminRemarkHasErrored(false));
+          dispatch(saveAdminRemarkSuccess(true));
+          dispatch(toastSuccess(SAVE_ADMIN_REMARK_SUCCESS, SAVE_ADMIN_REMARK_SUCCESS_TITLE));
+          dispatch(saveAdminRemarkIsLoading(false));
+          swal.close();
```
Comment (pveland621) [inline src/actions/remark.js:140]: wait this works lol?

**Review by pveland621 — APPROVED** [review]
only question was on the update_date on remark creation

File: src/Components/AdministratorPage/EditRemark/EditRemark.jsx (hunk 1 above)
Comment (scott062) [reply src/Components/AdministratorPage/EditRemark/EditRemark.jsx:63]: right it should be handled by system already

File: src/actions/remark.js (hunk 2 above)
Comment (scott062) [reply src/actions/remark.js:140]: yea bc the swal object is singular so technically its closing all swal

**Pull request merged**